src/traces/scattermapbox/hover.js

```diff
@@ -19,6 +19,8 @@ module.exports = function hoverPoints(pointData, xval, yval) {
         xa = pointData.xa,
         ya = pointData.ya;
 
+    if(cd[0].placeholder) return;
```

Comment:
we could.

Not sure `emptytrace` is better though. `tracewithnodatapoint` would be the most verbose. But I think `placeholder` does the trick as discussed in  https://github.com/plotly/plotly.js/pull/1004/files#diff-ad4f76ccd6044ed16514297078e13b84R1672